ALI/ALI.py

```diff
@@ -8,6 +8,7 @@
 from slicer.util import VTKObservationMixin
 import webbrowser
 import textwrap
+import pkg_resources
```

Comment:
I guess I overlooked this before since `pkg_resources` was not imported till now. `pkg_resources` is deprecated and should not be used.

It looks like the only usage is in `check_lib_installed`. You could perform the same task with `importlib.metadata`, something like:

```py
import importlib.metadata

...

installed_version = importlib.metadata.version(lib_name)
```

---

Re-creating this comment. I tried to add the comment to the `def check_lib_installed` line but GitHub made it like the change was requested on `import sys`.